Documentation/Security/GuidelinesIntegrators/Typoscript.rst.txt

```diff
@@ -33,17 +33,18 @@ The following code snippet gives an example:
      table = tt_content
      select {
        pidInList = 123
-       where = deleted=0
-       andWhere.data = GP:pageid
-       andWhere.wrap = uid=|
-       andWhere.intval = 1
+       where = deleted=0 AND uid=###PAGEID###
+       markers {
+           PAGEID.data = GP:pageid
+       }
      }
    }
 
-Without the :typoscript:`andWhere.intval = 1` instruction, the not sanitized
-argument passed by the `GET` / `POST` request `pageid` is being used for the
-database query. Specially formed `GET` / `POST` requests would cause an SQL
-injection.
+Argument passed by the `GET` / `POST` request `pageid` wrapped as markers are properly
+escaped and quoted to prevent SQL injection problems.
+
+See :ref:`TypoScript Reference <t3tsref:Functions/Select.html#markers>` for more
```

Comment:
I would like to use intersphinx using an anchor instead of url. This prevents 404 when url structure changes.

Reply:
@DanielSiepmann I pushed another commit resolving the cross-referencing issue.